sapl/templates/norma/normajuridica_form.html

```diff
@@ -58,29 +65,57 @@
          }
   });
 
-  var modal_estilos = 'display: block;'
-    +'width: 85%; max-width: 600px;'
-    +'background: #fff; padding: 15px;'
-    +'border-radius: 5px;'
-    +'-webkit-box-shadow: 0px 6px 14px -2px rgba(0,0,0,0.75);'
-    +'-moz-box-shadow: 0px 6px 14px -2px rgba(0,0,0,0.75);'
-    +'box-shadow: 0px 6px 14px -2px rgba(0,0,0,0.75);'
-    +'position: fixed;'
-    +'top: 50%; left: 50%;'
-    +'transform: translate(-50%,-50%);'
-    +'z-index: 99999999; text-align: center';
-
-  var fundo_modal_estilos = 'top: 0; right: 0;'
-    +'bottom: 0; left: 0; position: fixed;'
-    +'background-color: rgba(0, 0, 0, 0.6); z-index: 99999999;'
-    +'display: none;';
-
-  var meu_modal = '<div id="fundo_modal" style="'+fundo_modal_estilos+'">'
-    +'<div id="meu_modal" style="'+modal_estilos+'">'
-      +'<h2>Atenção! Ano de apresentação e ano da norma são diferentes.</h2><br />'
-      +'<button id="close_model_btn" type="button" class="btn btn-warning" data-dismiss="modal">'
-      +'Compreendo e quero continuar</button>'
-    +'</div></div>';
+  var modal_estilos = `
+    display: block;
+    width: 85%;
+    max-width: 600px;
+    background: #fff;
+    padding: 15px;
+    border-radius: 5px;
+    -webkit-box-shadow: 0px 6px 14px -2px rgba(0,0,0,0.75);
+    -moz-box-shadow: 0px 6px 14px -2px rgba(0,0,0,0.75);
+    box-shadow: 0px 6px 14px -2px rgba(0,0,0,0.75);
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%,-50%);
+    z-index: 99999999;
+    text-align: center;
+  `;
+
+  var fundo_modal_estilos = `
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    position: fixed;
+    background-color: rgba(0, 0, 0, 0.6);
+    z-index: 99999999;
+    display: none;
+  `;
+
+  var meu_modal = `
+    <div id="fundo_modal" style="${fundo_modal_estilos}">
+      <div id="meu_modal" style="${modal_estilos}">
+        <h2>Atenção! Ano de apresentação e ano da norma são diferentes.</h2><br />
+        <button id="close_model_btn" type="button" class="btn btn-warning" data-dismiss="modal">
+          Compreendo e quero continuar
+        </button>
+    </div>
+  </div>
+  `;
+
+  const confirmacao_mudanca_ementa_indexacao = `
+    <div id="id_fundo_confirmacao_mudanca_ementa_indexacao" style="${fundo_modal_estilos}">
+      <div id="id_confirmacao_mudanca_ementa_indexacao" style="${modal_estilos}">
+        <h3>Houve uma mudança de Matéria.</h3>
```

Comment:
```suggestion
        <h3>Houve a mudança de Matéria vinculada a norma.</h3>
```
(o "a" é com crase, não sei como coloca isso no MBP, kkkk)